docs/user_guide/backwards-compatibility.md

```diff
@@ -0,0 +1,44 @@
+# Our commitments to backwards compatibility
+
+We care deeply about minimizing negative impacts of changes to earthaccess, but we also care deeply about making earthaccess the most valuable it can be to our users. These are sometimes in conflict, and this documentation helps us make decisions that balance these needs in a way that's best for our users.
+
+## Our versioning scheme
+
+We use [SemVer](https://semver.org/) to tell you what to expect when upgrading. We recommend following the link to learn more, but here are the important ways this affects you:
+
+    1. There are 3 version parts: MAJOR.MINOR.PATCH (e.g. 1.2.3 is major version 1, minor release 2, patch release 3).
+    2. When the major version changes, anything can break! Always visit this documentation before doing a major upgrade.
+    3. When the minor version changes, new features should be available, but nothing should break. Visit the documentation to learn about new features.
+    4. When the patch version changes, only bugfixes should be included. Visit the CHANGELOG to learn more about the fixes.
+
+## Our commitments
+
+### Versioning
+
+We will follow SemVer. All version changes will consider [this](https://example.com/our-public-api) to be the public API documentation for the purposes of deciding whether a change is breaking or non-breaking.
+
+### Release Communication
+
+1. We will announce releases on the following channels: ___
+2. Release announcements will include a prominent notification of breaking changes, including a link to migration guide.
+
+### CHANGELOG
+
+1. We will update the CHANGELOG for every release
+2. The CHANGELOG will include prominent notification of breaking changes, including a link to migration guide.
+
+### Fixing Backwards Incompatible Changes
+
+1. We will plan to fix any backwards incompatible changes in non-major releases.
```

Comment:
```suggestion
1. We will plan to fix any backwards incompatible changes in non-major releases _or_ re-release with a major version bump.
```

I don't think we can necessarily commit to always doing this, sometimes it might be a mistake. I'm not sure. What's everyone else thing?